www/js/app.js

```diff
@@ -569,19 +602,38 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
      * @param {String} value The chosen content injection mode : 'jquery' or 'serviceworker'
      */
     function setContentInjectionMode(value) {
+        contentInjectionMode = value;
         if (value === 'jquery') {
-            if (isServiceWorkerReady()) {
```

Comment:
Agreed. I'll check what the effects are on Chromium of not disabling it first.

Reply:
OK, I've now checked, and we can't completely disable the Service Worker in case a user is accessing the app from an online implementation. In that case, it will now function as a PWA, and will need access to the Service Worker. This will be the case if a user accesses the implementation from https://kiwix.github.io/kiwix/kiwix-js, or (less likely) https://moz-extension.kiwix.org. However, I've added logic below to ensure that the ASSETS_CACHE is completely turned off and deleted. Note that a Chrome extension won't cache its own code in SW mode because it cannot use the Cache API, so we don't need to blacklist it explicitly.